.gitignore

```diff
@@ -0,0 +1 @@
+.vscode
```

Comment:
Why are ignoring it?

Reply:
I use other extensions that adds to it that is irrelevant to other contributors